include/usb/usb_common.h

```diff
@@ -65,10 +65,14 @@
 #define USB_INTERFACE_DESC		0x04
 #define USB_ENDPOINT_DESC		0x05
 #define USB_DEVICE_QUAL_DESC		0x06
+#define USB_OTHER_SPEED			0x07
+#define USB_INTERFACE_POWER		0x08
 #define USB_INTERFACE_ASSOC_DESC	0x0B
 #define USB_DEVICE_CAPABILITY_DESC	0x10
 #define USB_HID_DESC			0x21
 #define USB_HID_REPORT_DESC		0x22
+#define CS_INTERFACE			0x24
```

Comment:
Please rename to USB_CS_INTERFACE_DESC and USB_CS_ENDPOINT_DESC

Reply:
done